HTML - Personal Site/index.html

```diff
@@ -1,79 +1,65 @@
-          <p> I am a Scrum Expert and I 💗 learning new technlogy </p></td>
-      </tr>
-    </table>
-<hr>
-<h3>Work Experience</h3>
-<table cellspacing="10">
-  <thead>
-  <tr>
-    <th>Dates</th>
-    <th>Work</th>
-  </tr>
-</thead>
-<tbody>
-            <tr>
-                <td>2019-2021</td>
-                <td>ETX, Digital Experience, MassMutual</td>
-            </tr>
-            <tr>
-                <td>2018-2019</td>
-                <td>Wealth Management Technology, Morgan Stanley</td>
-            </tr>
-            <tr>
-                <td>2016-2017</td>
-                <td>Institutional Client, Sales Trading & Research, Dealogic</td>
-            </tr>
-            <tr>
-                <td>2014-2016</td>
-                <td>Wealth Management Technology, Morgan Stanley</td>
-            </tr>
-            <tr>
-              <td>2012-2014</td>
-              <td>Product & CLient Services, S&P Global</td>
-          </tr>
-  </tbody>
-  </table>
-  <hr>
-  <h3>Skills</h3>
-  <table cellspacing="10">
-    <tr>
-      <td>Web Development</td>
-      <td>⭐⭐⭐</td>
-    </tr>
-    <tr>
-      <td>iOS Development</td>
-      <td>⭐⭐</td>
-    </tr>
-    <tr>
-      <td>Painting</td>
-      <td>⭐⭐⭐</td>
-    </tr>
-    <tr>
-      <td>Photography</td>
-      <td>⭐⭐</td>
-    </tr>
-  </table>
-  <hr>
-<a href="hobbies.html">My Hobbies</a>
-<a href="contact-me.html">Contact Me</a>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <title>Madhavi Karra</title>
+  <link rel="stylesheet" href="css/styles.css">
+  <link href="https://fonts.googleapis.com/css?family=Merriweather|Montserrat|Sacramento" rel="stylesheet">
+  <link rel="icon" href="favicon.ico">
+</head>
+
+<body>
+  <div class="top-container">
+    <img class="top-cloud" src="images/cloud.png" alt="cloud">
+    <div class="title-text">
+      <h1>I'm Madhavi.</h1>
+      <h2>a <span class="pro">Scrum</span>Master</h2>
+    </div>
+    <img class="bottom-cloud" src="images/cloud.png" alt="cloud">
+    <img class="mountain" src="images/mountain.png" alt="mountain-img">
+  </div>
+
+  <div class="middle-container">
+    <div class="profile">
+      <img src="images/mk.png" alt="madhavi-profile-img">
+      <h2>Hello.</h2>
+      <p class="intro">I am currently on the R&D team working to help DX onboard HashiCorp Terraform. I ❤️ learning new technology.</p>
+    </div>
+    <hr>
+    <div class="skills">
+      <h2>My Skills.</h2>
+      <div class="skill-row">
+        <img class="code-img" src="https://media.giphy.com/media/3p50WGyUqeU5W/giphy.gif" alt="code-img">
+
+        <h3>Research & Development</h3>
+        <p>I used to be a developer a long long time ago and excited to get back in to coding and learning new skills</p>
+
+      </div>
+      <div class="skill-row">
+        <img class="chilli-img" src="images/chillies.png" alt="chillies-img">
```

Comment:
The 'alt' text of your images should be human readable. So for this one, instead of writing "chillies-img", you could write "Two red chillies". 

You can adjust the alt text on line 20, 25, 33, and this one.

Reply:
Thanks Allen, this is very helpful!